public/app/features/provisioning/Job/JobContent.tsx

```diff
@@ -29,6 +29,15 @@ export function JobContent({ job, isFinishedJob = false }: JobContentProps) {
             title={t('provisioning.job-status.status.title-job-completed-successfully', 'Job completed successfully')}
           />
         );
+      case 'warning':
+        return (
+          <Alert
+            severity="warning"
+            title={t('provisioning.job-status.status.title-warning-running-job', 'Warning running job')}
```

Comment:
```suggestion
            title={t('provisioning.job-status.status.title-warning-running-job', 'Job completed with warnings')}
```

Also `completed with errors` message in this case is a bit confusing, should probably say `completed with warnings`.